.github/workflows/pull_request.yml

```diff
@@ -76,6 +76,11 @@ jobs:
         cd infra
         ./gradlew cleanTest test --tests "*IT"
 
+    - name: Run DAO integration tests without cache in infra (backend)
```

Comment:
DAO-testit ajetaan omassa stepissään, että näkisi helpommin kumpi testiajo räjähtää (ylempi steppi ajaa ne kakku päällä)

Reply:
Hyvä huomio. Otin tuon `cleanTest`:in mukaan kun aiemmassakin oli tutkimatta sen kummemmin mitä se tekee, mutta sehän tosiaan poistaa aiemmat testiraportit. Huomasin nyt tätä korjatessa, että itse asiassa meillä on ollut tämä ongelma jo aiemmalta -- tuo IT-testien suorittama `cleanTest` on käynyt heivaamassa yksikkötestien testiraportit kuuseen. Luulisi että nekin halutaan kuitenkin ajaa, joten korjaan tuon tässä samalla.

Reply:
Tässä yhteydessä tuli tehtyä vähän isompikin taskipuljaus (tuli nyt epähuomiossa commitattua ja pushattua nuo näiden olemassolevien branchien päälle eikä tehtyä kokonaan uusia review-fiksibrancheja.) Käytännössä aiempi ongelma oli, että kun sekä IT:t että yksikkötestit ajettiin `test`-taskin kautta niin niiden outputit shadowasivat toisensa (ja test-taskin ajaminen tuntuu tyhjentävän output-hakemistot ilman `cleanTest`:iäkin mikäli siellä on jotain ennestään.) Täten tein seuraavaa:

1. Yksikkötestit ajetaan edelleen omassa taskissaan
2. Integraatiotestit ajetaan nyt omassa `integrationtest`-taskissaan. Tämä oli helpoin tapa saada niiden output omaan hakemistoonsa
3. Vastaavasti renamesin nyt tuon edellisen `test-without-cache`-taskin `integrationtest-without-cache`:ksi

Käytännössä näiden muutosten summana main-mergen jälkeen kaikkien kannattaa pullata myös env-repo, sillä taskien nimet muuttuivat.